zhmcclient/_adapter.py

```diff
@@ -194,6 +196,19 @@ def __init__(self, manager, uri, properties):
         """
         assert isinstance(manager, AdapterManager)
         super(Adapter, self).__init__(manager, uri, properties)
+        self._ports = None
+
+    @property
+    @_log_call
```

Comment:
While technically, this is part of the external API, we did not put the `@log_call` decorator on any of the other properties that are managers. Suggest to remove `@log_call`.